### PR TITLE
JAX-WS: Set eclipse settings on our beanvalidation fat to be 1.8 consistently.

### PR DESCRIPTION
**Pull request by neuwerk**
This PR sets the org.eclipse.jdt.core.prefs in `io.openliberty.jaxws.beanvalidation_fat` to Java 1.8. This should resolve some build issues related to this bucket's dependency on `java.lang.invoke.StringConcatFactory`

### CODE REVIEW

File: dev/io.openliberty.jaxws.beanvalidation_fat/bnd.bnd [hunk 1]
```diff
@@ -39,6 +39,7 @@ tested.features: \
   beanvalidation-3.0, \
   pages-3.1
 
+fat.minimum.java.level: 11
```
Comment (tkburroughs) [inline dev/io.openliberty.jaxws.beanvalidation_fat/bnd.bnd]: Also add:

javac.source: 11
javac.target: 11

**Review by tkburroughs — DISMISSED** [review]
In addition to also setting source/target in bnd.bnd.... please also update:
- .classpath to indicate JavaSE-11 instead of JavaSE-1.8
- org.eclipse.jdt.core.prefs and replace the 3 java 17 settings to java 11

**Review by jhanders34 — CHANGES_REQUESTED** [review]
The only change needed is updating the org.eclipse.jdt.core.prefs to be 1.8 and the eclipse problem goes away.  You do not need to change things to Java 11 for this FAT project.

**Review by jhanders34 — DISMISSED** [review]

**Comment by jhanders34** [comment]
#libby

**Review by jhanders34 — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Pull request merged**